index.html

```diff
@@ -851,25 +854,34 @@ <h2>
                 <ol>
                   <li>[=list/for each=] |file| of |files|
                     <ol>
-                      <li>Let |params:LaunchParams| be a new {{LaunchParams}}
-                      with {{LaunchParams/files}} set to a {{FrozenArray}} with
-                      a single element that is a {{FileSystemHandle}}
-                      corresponding to |file|.
+
+                      <li>Let |params:LaunchParams| be a new
+                      <a href="https://wicg.github.io/web-app-launch/#dom-launchparams">
+                      LaunchParams</a> with
+                      <a href="https://wicg.github.io/web-app-launch/#dom-launchparams-files">
+                      files</a> set to a {{FrozenArray}} with a single element
+                      that is a {{FileSystemHandle}} corresponding to |file|.
                       </li>
-                      <li>[=launch a web app=] with |params|.
+                      <li><a href="https://wicg.github.io/web-app-launch/#dfn-launching-a-web-application-with-handling">
+                      Launch a web application with handling</a> passing
+                      |manifest| and |params|.
                       </li>
                     </ol>
                   </li>
                 </ol>
               </li>
               <li>Else,
                 <ol>
-                  <li>Let |params:LaunchParams| be a new {{LaunchParams}} with
-                  {{LaunchParams/files}} set to a {{FrozenArray}} of
-                  {{FileSystemHandle}}s that correspond to the file paths named
-                  by |files|.
+                  <li>Let |params:LaunchParams| be a new
+                  <a href="https://wicg.github.io/web-app-launch/#dom-launchparams">
+                  LaunchParams</a> with
+                  <a href="https://wicg.github.io/web-app-launch/#dom-launchparams-files">
+                  files</a> set to a {{FrozenArray}} of {{FileSystemHandle}}s
+                  that correspond to the file paths named by |files|.
                   </li>
-                  <li>[=launch a web app=] with |params|.
+                  <li><a href="https://wicg.github.io/web-app-launch/#dfn-launching-a-web-application-with-handling">
```

Comment:
These ugly href links are due to web-app-launch not existing in specref yet. I've filed https://github.com/tobie/specref/issues/721 to try and work out how to fix that.